package.json

```diff
@@ -80,7 +80,7 @@
     "istanbul-middleware": "=0.2.2",
     "jsdoc": "=3.4.3",
     "jsdox": "=0.4.10",
-    "lisk-js": "=0.4.3",
+    "lisk-js": "git://github.com/LiskHQ/lisk-js#optional_data_test",
```

Comment:
@Tosch110 @SargeKhan could you explain the reason you aren't using a published verion of `lisk-js` ?